plugins/woocommerce/includes/class-wc-install.php

```diff
@@ -267,6 +267,13 @@ class WC_Install {
 	 */
 	const NEWLY_INSTALLED_OPTION = 'woocommerce_newly_installed';
 
+	/**
+	 * Option name used to track new installation versions of WooCommerce.
+	 *
+	 * @var string
+	 */
+	const INITIAL_INSTALL_VERSION_OPTION = 'woocommerce_initial_install_version';
```

Comment:
You might think about this already, but should we name the option as `woocommerce_initial_installed_version`?

Reply:
Great suggestion, done in https://github.com/woocommerce/woocommerce/pull/49139/commits/ac37ede0eef14e318b3cb825f89ada3c87845b55